src/main/java/net/unicon/lti/model/AlternativeDomain.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright 2021 Unicon (R)
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.unicon.lti.model;
+
+import org.apache.commons.lang3.StringUtils;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.util.Objects;
+
+@Entity
+@Table(name = "alternative_domain")
+public class AlternativeDomain extends BaseEntity {
```

Comment:
I would have suggested using Lombok's `@Data` but since this has already been deployed to the staging environment that change would cause way more trouble than it's worth at this point.

Reply:
Good point. Yes, better to keep it as it is.